tests/acceptance/29_simulate_mode/files_promises.cf.expected

```diff
@@ -0,0 +1,39 @@
+ warning: All changes in files will be made in the 'WORKDIR/state/PID.changes' chroot
```

Comment:
@vpodzime what do you think about this "basic shape" of the test?

I replace WORKDIR, PID and TIMESTAMP so that differences won't matter.

I can keep adding various files promises types, basically from the 28_inform_testing tests I wrote previously.

Reply:
:+1:  I don't have any better way up in my sleeve.